cartographer_ros/cartographer_ros/assets_writer_main.cc

```diff
@@ -150,7 +150,7 @@ void Run(const string& trajectory_filename, const string& bag_filename,
     rosbag::Bag bag;
     bag.open(bag_filename, rosbag::bagmode::Read);
     rosbag::View view(bag);
-    const ::ros::Time begin_time = view.getBeginTime();
+    const ::builtin_interfaces::msg::Time begin_time = view.getBeginTime();
```

Comment:
Actually, this particular file is an interesting case.  The changes that were done here are only because I did various 'sed' commands across all C++ files.  As-is, this has no chance of working because we need rosbag for this file.  So we could either revert back to the original, and just know it won't work completely, or we could leave some of these changes.  What's preferred?